portal/views/partials/address/address-tokens.njk

```diff
@@ -1,4 +1,4 @@
-<div class="block-section address-tokens">
+<div class="block-section address-tokens isDesktop">
```

Comment:
we already have a test for  isDesktop , if you have to use this css class  use underscore syntax like the rest of the css. i.e: is_desktop